### PR TITLE
Feature/cg 1189 support az nist encryption in transit rules

### PR DESCRIPTION
**Pull request by zhouse51**

### CODE REVIEW

File: src/aws/cis-1.2.0/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# [@cloudgraph/policy-pack-aws-cis-1.2.0-v0.12.1](https://github.com/cloudgraphdev/cloudgraph-policy-packs/compare/@cloudgraph/policy-pack-aws-cis-1.2.0@0.12.0...@cloudgraph/policy-pack-aws-cis-1.2.0@0.12.1) (2022-07-11)
```
Comment (tyler-dunkel) [inline src/aws/cis-1.2.0/CHANGELOG.md:1]: not sure why this is in here, can you rebase on alpha?

File: src/aws/cis-1.2.0/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@cloudgraph/policy-pack-aws-cis-1.2.0",
   "description": "Policy pack implementing CIS Amazon Web Services Foundations 1.2.0 Benchmark",
-  "version": "0.12.0",
+  "version": "0.12.1",
```
Comment (tyler-dunkel) [inline src/aws/cis-1.2.0/package.json:4]: 🤔 hmm maybe the repo wasnt fast forwarded from main?

File: src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+        condition {
+          allOf {
+            field
+            equals
+          }
+        }
+      }
+    }
+  }`,
+  resource: 'queryazureResourceGroup[*]',
```
Comment (tyler-dunkel) [inline src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts:142]: @aacotroneo is this a rule that will need to be rewritten once we have `auzureSubscription`?

File: src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+  severity: 'medium',
+  conditions: {
+    path: '@.activityLogAlerts',
+    array_any: {
+      and: [
+        {
+          path: '[*].region',
+          equal: 'global',
```
Comment (tyler-dunkel) [inline src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts:150]: are there any log alerts that arent global?

File: src/azure/nist-800-53-rev4/tests/nist-800-53-rev4-6.x.test.ts [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+import { Rule, Result, Engine } from '@cloudgraph/sdk'
+import 'jest'
+
+import Azure_NIST_800_53_61 from '../rules/azure-nist-800-53-rev4-6.1'
+import Azure_NIST_800_53_62 from '../rules/azure-nist-800-53-rev4-6.2'
+import Azure_NIST_800_53_63 from '../rules/azure-nist-800-53-rev4-6.3'
+import Azure_NIST_800_53_64 from '../rules/azure-nist-800-53-rev4-6.4'
+import { initRuleEngine } from '../../../utils/test'
+
+export interface SiteConfig {
```
Comment (tyler-dunkel) [inline src/azure/nist-800-53-rev4/tests/nist-800-53-rev4-6.x.test.ts:13]: are these exported types needed?

File: src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts (hunk 3 above)
Comment (aacotroneo) [reply src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts:142]: yes

File: src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts (hunk 4 above)
Comment (m-pizarro) [reply src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts:150]: Not really, we can simplify it and remove that condition

File: src/azure/nist-800-53-rev4/tests/nist-800-53-rev4-6.x.test.ts (hunk 5 above)
Comment (m-pizarro) [reply src/azure/nist-800-53-rev4/tests/nist-800-53-rev4-6.x.test.ts:13]: No, I will remove everything that is not used

**Review by tyler-dunkel — APPROVED** [review]

**Comment by tyler-dunkel** [comment]
@m-pizarro this should be good to go after merge conflict resolved

**Comment by m-pizarro** [comment]
> @m-pizarro this should be good to go after merge conflict resolved

@tyler-dunkel  done!

**Pull request merged**